web/js/select_column.js

```diff
@@ -0,0 +1,41 @@
+function SelectColumn(index, tableId) {
+    var columnSelector = '#' + tableId + ' tbody > tr > td:nth-child(' + (index + 1) + ')';
+    var cells = $(columnSelector);
+
+    // clear existing selections
+    if (window.getSelection) { // all browsers, except IE before version 9
+        window.getSelection().removeAllRanges();
+    }
+
+    if (document.createRange) {
+        cells.each(function(i, cell) {
+            var rangeObj = document.createRange();
+            rangeObj.selectNodeContents(cell);
+            window.getSelection().addRange(rangeObj);
+        });
+
+    // Internet Explorer before version 9
+    } else {
+        cells.each(function(i, cell) {
+            var rangeObj = document.body.createTextRange();
+            rangeObj.moveToElementText(cell);
+            rangeObj.select();
+        });
+    }
+}
+
+$(document).ready(function() {
```

Comment:
You can drastically simplify this: assign a class `select_header` to the columns you want to be selectable (right now you let select also the key column, which is not useful), and use
```JS
$(document).ready(function() {
    $('.select_header').click(function() {
        var columnNumber = $(this).index();
        var tableID = $(this).closest('table').attr('id');
        SelectColumn(columnNumber, tableID);
    });
});
```